nginx.conf

```diff
@@ -0,0 +1,17 @@
+server {
+  listen       80;
+
+  location / {
+    root /usr/share/nginx/html;
+    index index.html;
+    try_files $uri /index.html;
+  }
+  location /public/ {
```

Comment:
I think we don't need this block. The assets in `/usr/share/nginx/html` are already handled by the previous block.

Reply:
Oops, good catch. Borrowed an existing simple config.